doc/Makefile

```diff
@@ -46,14 +47,24 @@ html_dev:
 	@echo "Build finished. The HTML pages are in build/html."
 
 html-noplot:
-	$(SPHINXBUILD) -D plot_gallery=False -b html $(ALLSPHINXOPTS) build/html
+	$(SPHINXBUILD) -D plot_gallery=0 -b html $(ALLSPHINXOPTS) build/html
 	@echo
-	@echo "Build finished. The HTML pages are in build/html/stable."
+	@echo "Build finished. The HTML pages are in build/html."
```

Comment:
It shouldn't -- the comment about where it went was just incorrect before. Look above, I didn't change the output location.